test/runtime/samples/store-auto-subscribe-removed-store/_config.js

```diff
@@ -0,0 +1,19 @@
+import { writable } from '../../../../store';
+
+export default {
+	html: `
+		<p>undefined</p>
+	`,
+	async test({ assert, component, target }) {
+		component.store = writable('foo');
+		await Promise.resolve();
+		assert.htmlEqual(target.innerHTML, `
+			<p>foo</p>
+		`);
+		component.store = undefined;
+		await Promise.resolve();
+		assert.htmlEqual(target.innerHTML, `
+			<p>undefined</p>
+		`);
+	}
+};
```

Comment:
probably add another action to set the `component.store` back into a store to make sure it can turn from `undefined` back to a value

Reply:
Done!